src/inmanta/data/__init__.py

```diff
@@ -2666,7 +2788,9 @@ def to_dto(self) -> m.ResourceAction:
 class ResourceOrder(DatabaseOrder):
     """ Represents the ordering by which resources should be sorted"""
 
-    valid_sort_pattern: Pattern = re.compile("^(resource_type|agent|status|resource_id_value)\\.(asc|desc)$", re.IGNORECASE)
+    valid_sort_pattern: Pattern[str] = re.compile(
```

Comment:
I would prefer to simply declare a list of allowed fields, and derive everything from that.